### PR TITLE
Disable blynk and implement Prometheus exporter

### PR DESCRIPTION
**Pull request by vi7**

### CODE REVIEW

File: src/DHTDevice.h [hunk 1]
```diff
@@ -31,8 +35,24 @@ class DHTDevice: public Device {
 
         String status();
 
+        void collect(String *metrics);
```
Comment (Kristty) [inline src/DHTDevice.h:38]: really needed in interface?

File: src/WebServer.cpp [hunk 2]
```diff
@@ -36,6 +36,6 @@ void WebServer::registerEndpoint(Device *device, endpoint endpoints[], int size)
 
 void WebServer::sendReply(const String &content)
 {
-  this->sendHeader("Access-Control-Allow-Origin", "*");
+  this->enableCORS(true);
```
Comment (Kristty) [inline src/WebServer.cpp:39]: 👍🏾

File: src/DHTDevice.h (hunk 1 above)
Comment (vi7) [reply src/DHTDevice.h:38]: yes, its removal in the .h file causes the following err in the .cpp file: `inherited member is not allowedC/C++(298)`

File: src/WebServer.cpp (hunk 2 above)
Comment (vi7) [reply src/WebServer.cpp:39]: 😸

**Comment by vi7** [comment]
Closing this PR. It will be divided into 2 new PRs for blynk removal and Prom exporter implementation

**Comment by vi7** [comment]
Closing this PR. It will be divided into 2 new PRs for blynk removal and Prom exporter implementation

**Pull request closed without merging**